src/main/java/services/auth/AuthService.java

```diff
@@ -60,5 +66,18 @@ public static Claims validate(String authentication) throws Exception {
         //io.jsonwebtoken.SignatureException
         //IllegalArgumentException
     }
+
+    public static Response playerLogin(String quizCode) {
```

Comment:
Should probably just return the String and not the Reponse as it's the controller that should wrap it in a appropriate Response

Reply:
OK